main/main.cpp

```diff
@@ -2638,7 +2638,6 @@ Error Main::setup(const char *execpath, int argc, char *argv[], bool p_second_ph
 	}
 
 	GLOBAL_DEF_BASIC("internationalization/locale/include_text_server_data", false);
```

Comment:
I wonder if we should move this one to TestServer too for good measure.

Reply:
Probably, it's only used for `TextServerAdvanced` as well, since font settings were moved to TS, it makes sense to do it for the rest of TS related settings.